pakler.py

```diff
@@ -41,23 +42,28 @@
                    'len']
 MTD_PART_STRINGS = ['name',
                     'mtd']
-MTD_PART_SIZE = struct.calcsize(MTD_PART_FORMAT)
+MTD_PART_SIZE = struct.calcsize(MTD_PART_FORMAT)  # 76
 MTD_PART_COUNT = 10
 
 HEADER_FORMAT = "<III"
+HEADER_FORMAT_64 = "<QQQ"
 HEADER_FIELDS = ['magic',
                  'crc32',
                  'type']
 HEADER_CRC_OFFSET = 4  # Offset of the CRC in the file
+HEADER_CRC_OFFSET_64 = 8
 HEADER_HEADER_SIZE = struct.calcsize(HEADER_FORMAT)  # Size of the header's header (first 12 bytes)
+HEADER_HEADER_SIZE_64 = struct.calcsize(HEADER_FORMAT_64)  # First 24 bytes
 
 
-def calc_header_size(section_count, mtd_part_count=None):
+def calc_header_size(section_count, mtd_part_count=None, is64=False):
     """Return the calculated size of the header given a number of sections (and optionally of mtd_parts)."""
     if not mtd_part_count:
         mtd_part_count = section_count
 
-    return struct.calcsize(HEADER_FORMAT) + (section_count * SECTION_SIZE) + (mtd_part_count * MTD_PART_SIZE)
+    if is64:
+        return HEADER_HEADER_SIZE_64 + (section_count * SECTION_SIZE_64) + (mtd_part_count * MTD_PART_SIZE)
+    return HEADER_HEADER_SIZE + (section_count * SECTION_SIZE) + (mtd_part_count * MTD_PART_SIZE)
```

Comment:
Because what's the point of having `HEADER_HEADER_SIZE` if you can just use `struct.calcsize(HEADER_FORMAT)` directly! 😛

(Thanks for taking care of that).